src/triton_teleop/triton_teleop/sim_thrust_mapper.py

```diff
@@ -38,7 +38,7 @@ def thrust_allocator_callback(self, msg):
         else:
             for i in range(1,self.num_thrusters+1):
                 wrench_msg = Wrench()
-                wrench_msg.force.x = msg.data[i-1]
+                wrench_msg.force.z = msg.data[i-1]
```

Comment:
Why is this changed to z, if I look at the gazebo model, the x axes of the thruster cubes are seemingly the axes which the force should travel upon. Everything works but I'm wondering why this had to be changed?

Reply:
I reoriented the thruster model of the cube to match the orientation in the triton AUV model, they both point in the z direction now. I can change them to both point in x if you have a preference but I'm fine with either

Reply:
Ah you are right, I was accidentally on the wrong branch looking at the gazebo model, everything makes sense now